auth/realm/base/src/main/java/org/wildfly/security/auth/realm/FileSystemSecurityRealm.java

```diff
@@ -1160,22 +1352,95 @@ private void consumeContent(final XMLStreamReader reader) throws XMLStreamExcept
             }
         }
 
-        private boolean validNamespace(final String namespace) {
-            return ELYTRON_1_0.equals(namespace) || ELYTRON_1_0_1.equals(namespace);
+        private boolean validateDigitalSignature(String path, String name) throws IllegalStateException {
+            try {
+                DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
+                dbf.setNamespaceAware(true);
+                Document doc = dbf.newDocumentBuilder().parse(path);
+                NodeList nl = doc.getElementsByTagNameNS(XMLSignature.XMLNS, "Signature");
+                if (nl.getLength() == 0) {
+                    throw new RealmUnavailableException("Cannot find Signature element");
```

Comment:
This message should be in ElytronMessages.

Reply:
Moved to ElytronMessages